project/index.js

```diff
@@ -218,13 +219,40 @@ const sexReligionAll = Promise.all([
   dimensions.push(dimension)
 })
 
+
+const pricesBasedOnLocationPromise = d3.csv('data/median-resale-prices-for-registered-applications-by-town-and-flat-type/median-resale-prices-for-registered-applications-by-town-and-flat-type.csv', function(d) {
+    return {
+       quarter: d.quarter,
+       town: d.town,
+       flat_type: d.flat_type,
+       price: d.price
+    }
+})
+
+
+const priceInfoAll = Promise.all([
+  pricesBasedOnLocationPromise
+]).then(function(data) {
+  var dimension = "prices"
```

Comment:
price isn't a dimension. The dimensions are demographics dimensions used to recommend district + housing type.

Reply:
Removed it. Probably not necessary for prices?